contracts/pre-propose/dao-pre-propose-approver/src/tests.rs

```diff
@@ -14,7 +14,7 @@ use dao_pre_propose_approval_single::{
     state::PendingProposal,
 };
 use dao_pre_propose_base::{error::PreProposeError, msg::DepositInfoResponse, state::Config};
-use dao_proposal_single as cps;
+use dao_proposal_single as dps;
```

Comment:
I would put cps to dps change into a dedicated PR which will make this PR smaller and more readable. Happy to handle it if you don't mind.

Reply:
Please! A separate PR for this would be nice.

Reply:
Done, see #757

Reply:
Merged! Thank you!